shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class NetSimApiTest < Minitest::Unit::TestCase
+
+  def create_record(record)
+    @net_sim_api.post "/v3/netsim/#{@shard_id}/#{@table_name}", record.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
+    @net_sim_api.last_response.location.split('/').last
+  end
+
+  def read_records
+    @net_sim_api.get "/v3/netsim/#{@shard_id}/#{@table_name}"
+    JSON.parse(@net_sim_api.last_response.body)
+  end
+
+  def update_record(id, record)
+    @net_sim_api.put "/v3/netsim/#{@shard_id}/#{@table_name}/#{id}", record.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
+    JSON.parse(@net_sim_api.last_response.body)
+  end
+
+  def delete_record(id)
+    @net_sim_api.delete "/v3/netsim/#{@shard_id}/#{@table_name}/#{id}"
+  end
+
+  def test_create_read_update_delete
+    # The Tables API does not need to share a cookie jar with the Channels API.
```

Comment:
Corrected in the ["make test helpers private" commit](https://github.com/code-dot-org/code-dot-org/commit/80d5715d19ef3e07fd0d5937ba23c0812dd472d4).